conda/shell/condabin/_conda_activate.bat

```diff
@@ -29,7 +29,7 @@
 @REM This method will not work if %TMP% contains any spaces.
 @FOR /L %%I IN (1,1,100) DO @(
     SET UNIQUE_DIR=%TMP%\conda-!RANDOM!
```

Comment:
```suggestion
    SET "UNIQUE_DIR=%TMP%\conda-!RANDOM!"
```

Shouldn't this be quoted too? See https://stackoverflow.com/a/67542742/3407590 for quoted `SET` instructions.